run.go

```diff
@@ -939,6 +939,13 @@ func (b *Builder) configureEnvironment(g *generate.Generator, options RunOptions
 		}
 	}
 
+	for _, envSpec := range []string{"http_proxy", "https_proxy", "no_proxy"} {
```

Comment:
Need an option --http-proxy, which is defaulted to true to turn this off in buildah run and buildah bud.

buildah bud --http-proxy=false ...

Reply:
Also I thought you wanted to test upper case.

Reply:
I've pushed an update that includes more proxy vars, including upper case.  The initial push was the subset vars that I've found that work in the most common cases.

Still needs an option, which I haven't figured out how to do yet.